templates/unicorn.rb

```diff
@@ -4,9 +4,9 @@
 timeout (ENV["UNICORN_TIMEOUT"] || 15).to_i
 preload_app true
 
-before_fork do |server, worker|
+before_fork do |_, _|
```

Comment:
Thoughts on going with `_server`, `_worker` instead? Might be nice in the future to know what these variables were.